code/scripting/api/libs/graphics.cpp

```diff
@@ -1157,6 +1216,8 @@ ADE_FUNC(drawTargetingBrackets, l_Graphics, "object Object, [boolean draw=true,
 		line_draw_list.flush();
 	}
 
+	gr_lua_screen.active = false;
```

Comment:
This should come after the `g3_end_frame()` below to match its other usages. I don't think it happens to matter in this case, but it could in the future.

Reply:
Good call. I've addressed this and I think everything else.